### PR TITLE
02 06 2023 ship search all and teams message to v1 dot0

### PR DESCRIPTION
**Pull request by yiwenwang-ms**

### CODE REVIEW

**Comment by Jarbas-MSFT** [comment]
Hello @yiwenwang-ms, thank you very much for your PR! I reviewed your PR and left one comment. Please, take a look when you will have time.

File: changelog/Microsoft.Search.json [hunk 1]
```diff
@@ -1507,6 +1507,72 @@
       "CreatedDateTime": "2022-12-01T00:00:00",
       "WorkloadArea": "Search",
       "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
```
[the remark this answers is not in the document]
Comment (yiwenwang-ms) [reply changelog/Microsoft.Search.json:1513]: Hi Jarbas, thanks for reviewing the PR, do you mean chatMessage?
It only added a new link in the search api overview.

This is the previous PR, for new entity type, it only be added into the concepts, and it is shared for both beta and v1.0.
https://github.com/microsoftgraph/microsoft-graph-docs/pull/17686/files#diff-82124a8c0c82be7bfcdb68e839f881957e866dfaf8694402d2a3a79151bcdf3f

[the remark this answers is not in the document]
Comment (Jarbas-MSFT) [reply changelog/Microsoft.Search.json:1513]: In case of just a link reference, we can remove this entry.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review.

**Pull request merged**